build_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -40,9 +40,9 @@ class DevCompilerBuilder implements Builder {
     try {
       await createDevCompilerModule(module, buildStep);
     } on DartDevcCompilationException catch (e) {
-      log.warning('Error compiling ${module.jsId}:\n$e');
       await buildStep.writeAsString(
           buildStep.inputId.changeExtension(jsModuleErrorsExtension), '$e');
+      log.severe('', e);
```

Comment:
Empty string? Weird?

Reply:
This is just logging the error - it feels a bit weird for sure but there is already additional context provided by the loggers name (it is named based on the current builder and primary input).

The logger handles consistently formatting the error already as well - so that ends up being nicer than passing the error as the message.

Reply:
ack

Reply:
...why no stack? Not useful?

Reply:
Ya - we don't have a true stack trace here. This error is originating from a worker, and contains the entire message that the worker sent back - which may or may not contain a useful stack trace already.